### DISCUSSION TITLE
Save states

### DISCUSSION THREAD

**Discussion by ZahierRei** [Vdauphin/HeartsAndMinds#1629] [Question and answer] [answered]
Questions how do i add delete and load Save state .. 
as now i can save to test but cant delete it when I'm in the editor .
ps this is for init (editor)

**Comment by Vdauphin** [comment, marked as answer]
Hello,
You can't save in editor
You can still execute in debug console:
```sqf
[] call btc_db_fnc_delete;
```

Cheers

**Reply by ZahierRei** [reply]
If i want to add it as an option in the item init similar to the save script you gave previously . 
If possible i want to add the 
- delete save state
- load save state 
This is on top of the save script you help me with previously

**Reply by Vdauphin** [reply]
See https://github.com/Vdauphin/HeartsAndMinds/discussions/1626

```sqf
 _action = ["deletemygame", "Delete", "", {[] remoteExecCall ["btc_db_fnc_delete", 2]}, {true}] call ace_interact_menu_fnc_createAction;
[this, 0, ["ACE_MainActions"], _action] call ace_interact_menu_fnc_addActionToObject;
```

**Reply by ZahierRei** [reply]
thank you so much it works .
